### PR TITLE
PRESS4-303 | Payment Section

### PR DESCRIPTION
**Pull request by sumathi3399**
JIRA link:- https://jira.newfold.com/browse/PRESS4-303

Added dedicated payment section in store page.
when we click on payments tab, we can see all the payments options available for the user.

Screenshot 2023-08-08 at 12 49 50 PM

### CODE REVIEW

File: src/components/AllPayments.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+      <form
+        id="payment-details"
+        onSubmit={async (event) => {
+          event.preventDefault();
+          event.stopPropagation();
+          const payload = {
+            ...Object.fromEntries(new FormData(event.target).entries()),
+            ...formChanges,
+          };
+          console.log(payload);
```
Comment (aratidgr8) [inline src/components/AllPayments.js:87]: Please remove

File: src/components/AllPayments.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+                }
+              }
+            }
+            await updatePaymentMethods.trigger([
+              addedGateways,
+              removedGateways,
+            ]);
+          }
+          if (isFormDirty.razorpay) {
+            await settings.trigger({
```
Comment (aratidgr8) [inline src/components/AllPayments.js:120]: These texts might need translations

Comment (sumathi3399) [reply src/components/AllPayments.js:116]: done

File: src/components/AllPayments.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
+import { useEffect, useState } from "@wordpress/element";
```
Comment (aratidgr8) [inline src/components/AllPayments.js:1]: Organize imports

Comment (sumathi3399) [reply src/components/AllPayments.js]: done

File: src/components/AllPayments.js [hunk 4]
```diff
@@ -0,0 +1,190 @@
+import { useState } from "@wordpress/element";
```
Comment (aratidgr8) [inline src/components/AllPayments.js:1]: Organize imports

Comment (aratidgr8) [reply src/components/AllPayments.js:1]: @sumathi3399  Plz ignore

File: src/components/Payment.js [hunk 5]
```diff
@@ -1,174 +1,59 @@
...
-          values={values}
-        />
-      </Section.Settings>
-    </Section.Content>
+
+      <Section.Content>
+        <Section.Settings title={__("Offline Payments", "wp-module-ecommerce")}>
+          <div className="yst-border yst-rounded-md yst-p-6">
+            <CheckboxGroup
+              id="woocommerce_toggle_gateway_enabled"
```
[the remark this answers is not in the document]
Comment (sumathi3399) [reply src/components/Payment.js:30]: done

**Review by aratidgr8 — APPROVED** [review]

**Comment by sumathi3399** [comment]
closing this PR, since it is replaced with other PR https://github.com/newfold-labs/wp-module-ecommerce/pull/113

**Pull request closed without merging**